R/calc_survival_probability_merton.R

```diff
@@ -2,8 +2,8 @@
 #'
 #' Function calculates survival probability for a maturity based on a structural
 #' Merton model.
-#' For details on implementation please compare [CreditRisk::Merton()].
-#' Unlike [CreditRisk::Merton()] this implementation:
+#' For details on implementation please compare `CreditRisk::Merton()`.
+#' Unlike `CreditRisk::Merton()` this implementation:
```

Comment:
We don't use the package CreditRisk. If we `[link it]`, R CMD check will expect us to Import: or Suggest it.